libraries/AP_Beacon/AP_Beacon_Nooploop.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "AP_Beacon_Backend.h"
+
+#define NOOPLOOP_MSG_BUF_MAX      256
+
+#define NOOPLOOP_HEADER                     0x55    // message header
```

Comment:
Please move whatever `#define`s you don't need in the header into the cpp file.  Nobody else needs to know they exist...